crates/daphne/src/vdaf/mod.rs

```diff
@@ -36,9 +43,13 @@ pub use self::mastic::MasticWeightConfig;
 #[derive(Debug, thiserror::Error)]
 pub(crate) enum VdafError {
     #[error("{0}")]
-    Codec(#[from] CodecError),
+    Codec(#[from] prio_09::codec::CodecError),
+    #[error("{0}")]
+    Vdaf(#[from] prio_09::vdaf::VdafError),
+    #[error("{0}")]
+    CodecLatest(#[from] prio_latest::codec::CodecError),
```

Comment:
I see the logic in this, but adding these variants makes the code harder to read. Two suggestions:
* Add comments explaining the purpose of the variants
* Our naming convention should be that the legacy thing gets an odd name, and the new thing gets the normal name. For example:
```suggestion
    CodecDraft09(#[from] prio_09::codec::CodecError),
    #[error("{0}")]
    VdafDraft09(#[from] prio_09::vdaf::VdafError),
    #[error("{0}")]
    Codec(#[from] prio_latest::codec::CodecError),
```
Optionally, you might add "Latest" the latest variants.